client/my-sites/menus/main.jsx

```diff
@@ -104,28 +104,37 @@ var Menus = React.createClass( {
 				)
 			},
 			associationChanged: {
-				menu: function( menu, location ) {
+				menu: function( mainMenu, mainLocation ) {
```

Comment:
I was renaming them because eslint was throwing warnings. I think it might be a good idea to keep them but that might just be my preference.